sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/LiveMetricsMetricExporter.cs

```diff
@@ -31,46 +30,16 @@ public override ExportResult Export(in Batch<Metric> batch)
         {
             var list = new List<Models.MetricPoint>(capacity: (int)batch.Count); // TODO: POSSIBLE OVERFLOW EXCEPTION (long -> int)
 
-            //Debug.Assert(batch.Count == 11);
             foreach (var metric in batch)
             {
                 foreach (ref readonly var metricPoint in metric.GetMetricPoints())
                 {
-                    //Debug.WriteLine(LiveMetricConstants.Mappings[metric.Name]);
-
-                    switch (metric.MetricType)
+                    if (DataTranslator.TryConvertOTelToAzure(metric, metricPoint, out var azureMetricPoint))
```

Comment:
Do we need a method and return variable allocation here.  There are very less or no chance of this method returning false.

Reply:
Would you be okay with a TODO here?
I don't want to make major changes because all of this is likely to go away.